pkg/domain/podman/credential_handler.go

```diff
@@ -16,6 +17,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var credentialMountInContainer = map[string]string{
+	types.ClaimsServerSecret:  "/etc/service-controller/certs/",
```

Comment:
Why do we have a separate cert for claims and site-server in podman?

Reply:
The claims server cert is not yet being used. Better to remove it and just add when claims become
supported?